src/components/backend-ai-storage-list.ts

```diff
@@ -860,12 +861,16 @@ export default class BackendAiStorageList extends BackendAIPage {
           <div style="margin: 10px 0px">${_t('data.explorer.Permissions')}</div>
           <div style="display: flex; justify-content: space-evenly;">
             <wl-label>
```

Comment:
I understand and agree with your opinion. I searched about both two libraries (weightless and mwc) and I checked the weightless latest release year is 2019. I'll replace weightless to mwc-radio and request review again. 🙏

Reply:
I applied `--general-textfield-selected-color` color to Share Folder panel's permission radio inputs because there's no color for radio input. Please check the blow image and give your opinion.

<img width="417" alt="image" src="https://user-images.githubusercontent.com/5039744/177352804-4c21e8c5-fac9-467b-af79-5d3b85afc599.png">